objects/Game.pm

```diff
@@ -114,12 +114,12 @@ sub new
     # Also add a space after the : in case there was no
     # space between the : and the rack
     $line =~ s/\s*:/: /g;
-    
+
```

Comment:
Try to avoid changing whitespace like this. It makes it harder to determine what changes "you" made as opposed to the automatic formatting of whatever text editor you happen to be using.